duecredit/tests/test_api.py

```diff
@@ -150,7 +152,12 @@ def test_noincorrect_import_if_no_lxml(monkeypatch):
         # script with decorated funcs etc -- should be importable
         {'script': stubbed_script}
     ])
-def test_noincorrect_import_if_no_lxml_numpy(monkeypatch, kwargs, env, stubbed_env):
+def test_noincorrect_import_if_no_lxml_numpy(
+    monkeypatch: 'MonkeyPatch',
+    kwargs,
```

Comment:
These arguments aren't annotated.  I suggest adding `allow_incomplete_defs = False` to the project's mypy configuration so that such problems are caught.

Reply:
I m working with `mypy --strict` that list 220-something todo ... these items are not forgtoten